src/Microsoft.AspNet.Authentication.Cookies/CookieAuthenticationHandler.cs

```diff
@@ -149,9 +160,9 @@ private CookieOptions BuildCookieOptions()
 
             var cookieValue = Options.TicketDataFormat.Protect(model);
 
-            if (model.Properties.IsPersistent)
+            if (model.Properties.IsPersistent && _renewExpiresUtc.HasValue)
```

Comment:
The use of .HasValue is inconsistent in this code. In other areas == null is checked.
Should be consistent.